AttackItem.h

```diff
@@ -1 +1,10 @@
-#include "Item.h"
\ No newline at end of file
+#pragma once
+
+#include "Item.h"
+#include "Character.h"
```

Comment:
You can forward declare `Character` since you aren't using the type in this class.
it's being used in the cpp file though, which is where `Character.h` should be included
- [ ] revise